nautobot/extras/models/customfields.py

```diff
@@ -161,6 +163,55 @@ def get_custom_fields(self, advanced_ui=None):
         # 2.0 TODO: #824 field.slug rather than field.name
         return OrderedDict([(field, self.cf.get(field.name)) for field in fields])
 
+    def get_custom_field_groupings_basic(self):
+        """
+        This method exists to help call get_custom_field_groupings() in templates where a function argument (advanced_ui) cannot be specified.
+        Return a dictonary of custom fields grouped by the same grouping in the form
+        {
+            <grouping_1>: [cf1, cf2...],
+            ...
+            <grouping_5>: [cf8, cf9...],
+            ...
+        }
+        which have advanced_ui set to False
+        """
+        return self.get_custom_field_groupings(advanced_ui=False)
+
+    def get_custom_field_groupings_advanced(self):
+        """
+        This method exists to help call get_custom_field_groupings() in templates where a function argument (advanced_ui) cannot be specified.
+        Return a dictonary of custom fields grouped by the same grouping in the form
+        {
+            <grouping_1>: [cf1, cf2...],
+            ...
+            <grouping_5>: [cf8, cf9...],
+            ...
+        }
+        which have advanced_ui set to True
+        """
+        return self.get_custom_field_groupings(advanced_ui=True)
+
+    def get_custom_field_groupings(self, advanced_ui=None):
+        """
+        Return a dictonary of custom fields grouped by the same grouping in the form
+        {
+            <grouping_1>: [cf1, cf2...],
+            ...
+            <grouping_5>: [cf8, cf9...],
+            ...
+        }
+        """
+        record = {}
+        fields = CustomField.objects.get_for_model(self)
+        if advanced_ui is not None:
+            fields = fields.filter(advanced_ui=advanced_ui)
+
+        for field in fields:
+            data = (field, self.cf.get(field.name))
+            record.setdefault(field.grouping, []).append(data)
+        record = OrderedDict(sorted(record.items()))
```

Comment:
Given dicts are order-preserving in modern Python, does this work just as well if you use `dict(sorted(record.items()))` instead?